### DISCUSSION TITLE
A page belonging to a panel with $shouldRegisterNavigation set to false throws a javascript error in the console

### DISCUSSION THREAD

**Discussion by hotsaucejake** [filamentphp/filament#10880] [Help] [answered]
I have a page belonging to a panel.  I don't want the navigation to this page to show up in the main sidebar, so I set `protected static bool $shouldRegisterNavigation = false;` in the Page class and everything works as expected.

However, looking at the console, I get this error: Uncaught TypeError: activeSidebarItem is null

I'm assuming it's looking for the activeSidebarItem in order to show it as active, however it's not there because I disabled the navigation.  

Is there something else I should set to prevent this or is this a bug?

**Comment by hotsaucejake** [comment, marked as answer]
Nvm, this is fixed now with the latest update.

**Reply by leandrocfe** [reply]
Close the topic please
